.test-infra/jenkins/job_beam_PreCommit_Go_GradleBuild.groovy

```diff
@@ -30,7 +30,7 @@ job('beam_PreCommit_Go_GradleBuild') {
   common_job_properties.setTopLevelMainJobProperties(
     delegate,
     'master',
-    240)
```

Comment:
We had to increase it in the past from 150 to 240 min because Jenkins would sometimes timeout while to making a slow progress through the suite. Perhaps the infrastructure is in better shape now but maybe we can introduce/decrease a per-test limit instead?

Reply:
I think in those cases its easier to bump up the timeout when it happens then to have our build servers constantly building.